mathics/doc/common_doc.py

```diff
@@ -769,27 +780,30 @@ def get_tests(self) -> Iterator[Tests]:
                     index = 1
                     for doctest in section.items:
                         if isinstance(doctest, DocTests):
-                            doctest.index = index
-                            doctest.chapter = self.chapter.title
-                            doctest.part = self.chapter.part.title
-                            doctest.section = self.title
-                            doctest.key = (
-                                doctest.part,
-                                doctest.chapter,
-                                doctest.section,
+                            # Why monkey patch DocTests?
```

Comment:
In this branch I reverted these changes. And "make doctest" still works. 

I suspect that your suspicion that `Tests` and `DocTests` can be combined. I am not seeing `Tests` used in either `mathics/doc/latex_doc.py` or `mathics_django/docpipeline.py`. So I don't know right now why we need this.

However let's old off of making this change until we get some testing in place.

Reply:
@rocky, this change aimed to determine if these attributes were used in any other place. And I can report that they are not used. Now, I am studying how the class hierarchy works. When I finish with it, I will try to improve the documentation in the common_doc.py module, and start adding some pytests. In the meantime, feel free to pick the parts from this PR that you find useful.

Reply:
Ok. If you are going to start writing pytests, I'll hold off. (I have been procrastinating on this). There are other things that I have backlogged (which don't have anything to do with Mathics3).